openpype/hosts/resolve/api/workio.py

```diff
@@ -27,14 +27,16 @@ def save_file(filepath):
     pm = get_project_manager()
     file = os.path.basename(filepath)
     fname, _ = os.path.splitext(file)
+    dname, _ = fname.split("_v")
```

Comment:
This is NOT safe. OpenPype doesn't require `_v` to be an existing prefix. Similarly, if a filename was named `myfile_variant_v001` this would error too. Do **not** do this! :)

I'm seeing now `dname` is also used in the logic below in `open_file`, but this looks like issues just waiting to happen!